components/SearchBar.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+export default function SearchBar(props) {
+  return (
+    <div>
+      <form>
+        <label htmlFor="searchBar">
+          <input
+            type="text"
+            placeholder="what are you searching for?"
+            onChange={props.searchFunction}
+          ></input>
```

Comment:
Input is most commonly self-closing (since it cannot have any content in HTML): `<input />`